docs/getting_started.rst

```diff
@@ -160,11 +164,13 @@ Install Node.js, Yarn and other dependencies
 
 The Python project-specific dependencies installed above will install ``nodeenv``, which is a useful tool for using specific versions of Node.js and other Node.js tools in Python environments. To setup Node.js and Yarn within the Kolibri project environment, ensure your Python virtual environment is active, then run:
 
+For a more detailed guide to using nodeenv see :doc:`/howtos/nodeenv`.
+
 .. code-block:: bash
 
   # node.js, npm, and yarn
   nodeenv -p --node=16.13.2
```

Comment:
This line looks like an older version that is now superseded by [this one in the ``nodenv`` howto guide](https://github.com/learningequality/kolibri/pull/9592/files#diff-60c0249794f30f55e92658d4326576ac7ad9d06574246bcb4c1ffaf641fbb16cR27) 🤔